demo/lib/distro.bash

```diff
@@ -445,12 +447,27 @@ fedora-config-containerd-post() {
     fi
 }
 
+fedora-disable-swap() {
+    # Using swapoff is not enough as we also need to disable the swap from systemd
+    # and then reboot the VM
+    if vm-command "swapon --show | grep -q partition"; then
+	echo "Disabling swap"
+	vm-sed-file /etc/fstab '/^\/.*[[:space:]]swap[[:space:]].*$/d'
+	vm-command "systemctl --type swap"
+	vm-command "for swp in \`systemctl --type swap | awk '/\\.swap/ { print \$1 }'\`; do systemctl stop \"\$swp\"; systemctl mask \"\$swp\"; done"
+	vm-command "swapoff --all"
+	vm-reboot
+    fi
+}
+
```

Comment:
nitpick: couldn't/shouldn't this be better done in `fedora-bootstrap-commands*()` which is always run on the first boot ?

Reply:
Indeed, the `default-bootstrap-commands()` even sets the ip_forward already. I am just not seeing it set in my test setup, need to investigate why the default func is not run.

Reply:
Yep, will do